docker.gemrc.example

```diff
@@ -0,0 +1,9 @@
+---
+:backtrace: false
+:verbose: true
+:bulk_threshold: 1000
+:update_sources: true
+:sources:
+  - http://172.17.0.1:8808
```

Comment:
Do you run a gem host locally? Please comment out those lines and add some kind of an extra comment to clarify the need for those lines. I think it's really worth to keep, but make the reasons to have it more clearer.